strutil/strutil.go

```diff
@@ -203,3 +203,19 @@ func ElliptLeft(str string, n int) string {
 
 	return "…" + string(rstr[len(rstr)-n+1:])
 }
+
+// ListsSame verifies that two lists of strings have strictly the same strings,
+// in the same order.
+func ListsSame(l1, l2 []string) bool {
```

Comment:
After looking at #9097, maybe the name should be `ListsAreSame` or `ListsEqual`?

Reply:
yes, the name is a bit odd vs conventions

Reply:
this should be called either ListEqual or ListsEqual,  I think both are ok with conventions I see, I prefer the former

Reply:
Renamed to `ListEqual()`